commons/tool/src/webapp/WEB-INF/bootstrap.jsp

```diff
@@ -41,7 +41,9 @@
         <div id="Mrphs-sakai-commons" class="portletBody commons-portletBody">
 
             <ul id="commons-toolbar" class="navIntraTool actionToolBar hidden" role="menu"></ul>
-
+            <div class="page-header">
+                <h1>Commons</h1>
+            </div>
             <div id="commons-main-container">
```

Comment:
I think this should be removed until we discuss it further. The tool header was purposefully kept out of the commons tool as it made the layout in multi-tool pages look clunky. Also, it probably needs to be configurable, either in properties or via i18n.